### PR TITLE
Decompose Swish to Sigmoid + Prod

### PR DESCRIPTION
**Pull request by rvyunov**
It is workaround to restore performance for efficientnet-* models.

### CODE REVIEW

**Comment by rvyunov** [comment]
@elatkin , @andrejsokolov , @nikita-kud , please take a look at this changes.

**Review by elatkin — APPROVED** [review]

File: inference-engine/tests/functional/plugin/myriad/single_layer_tests/swish.cpp [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+#include <tuple>
+#include <string>
+
+using Parameters = std::tuple<
+        ngraph::element::Type,
+        ngraph::Shape,
+        LayerTestsUtils::TargetDevice>;
+
+namespace LayerTestsDefinitions {
+class SwishLayerTest : public testing::WithParamInterface<Parameters>,
```
Comment (Maxim-Doronin) [inline inference-engine/tests/functional/plugin/myriad/single_layer_tests/swish.cpp:19]: Swish has been already presented in `inference-engine/tests/functional/plugin/myriad/shared_tests_instances/single_layer_tests/activation.cpp`

Comment (nikita-kud) [inline inference-engine/tests/functional/plugin/myriad/single_layer_tests/swish.cpp:19]: Could you explain, what different between this test and test from the `activation.cpp` file ?

**Review by andrejsokolov — APPROVED** [review]

File: inference-engine/tests/functional/plugin/myriad/single_layer_tests/swish.cpp (hunk 1 above)
Comment (rvyunov) [reply inference-engine/tests/functional/plugin/myriad/single_layer_tests/swish.cpp:19]: Good catch! Thx.

Comment (rvyunov) [reply inference-engine/tests/functional/plugin/myriad/single_layer_tests/swish.cpp:19]: Nothing. Thx, just a mistake.

Comment (rvyunov) [reply inference-engine/tests/functional/plugin/myriad/single_layer_tests/swish.cpp]: Fixed

Comment (rvyunov) [reply inference-engine/tests/functional/plugin/myriad/single_layer_tests/swish.cpp]: Fixed

**Review by nikita-kud — APPROVED** [review]

**Review by Maxim-Doronin — APPROVED** [review]

**Comment by rvyunov** [comment]
@Maxim-Doronin , could you merge it, please?

**Pull request merged**